.github/workflows/pr.yml

```
@@ -37,6 +37,12 @@ jobs:
     name: Lint
     uses: ./.github/workflows/lint.yml
 
+  generate-legacy-subgraph:
+    name: Generate Legacy Subgraph Schema
+    uses: ./.github/workflows/generate-legacy-subgraph.yml
```

actionlint (1.7.4)

could not read reusable workflow file for "./.github/workflows/generate-legacy-subgraph.yml": open /home/jailuser/git/.github/workflows/generate-legacy-subgraph.yml: no such file or directory

(workflow-call)